src/main/groovy/io/seqera/wave/controller/ContainerTokenController.groovy

```diff
@@ -251,6 +251,10 @@ class ContainerTokenController {
                 scanId,
                 ip,
                 offset)
+        if( req.containerConfig.labels ){
```

Comment:
with the `labels` now in the `containerConfig` attribute, I think we don't need changes to this class any more